clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -183,6 +209,10 @@ def deploy_model(self, name, version, input_type, image, num_replicas=1):
                                 create_model_container_label(name, version),
                                 CLIPPER_DOCKER_LABEL:
                                 ""
+                            },
+                            'annotations': {
+                                "prometheus.io/scrape": "true",
+                                "prometheus.io/port": "1390"
```

Comment:
Are these Clipper-specific, or is this something prometheus will automatically pick up?

Reply:
This is something prometheus will automatically pick up. I documented them here: https://github.com/simon-mo/clipper/tree/kube_metrics/monitoring#scrape-target-discovery